web/templates/pages/member_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}TrackCat Member{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member_profile">
+			<div class="col-md-12">
+			<h2>{% if user.fullname %}
+			{{ user.fullname }} 
+			{% else %}
+			{{ user.username }}
+			{% endif %}'s profile</h2>
+			{% for field in form %}
+			{{ field.error }}
+			{% endfor %}
+			</div>	
+		</div>
+	</div>
+	<div class="row">
+		<div class="col-md-4 col-xs-6">
+			<img src="../../static/img/{{user.profile_picture}}">
```

Comment:
good question :) I can do that the photo is obligatory when opening a new profile :)  or I can write 'if stuff' as above in h2 and add a blank photo like FB has. Will do it. :+1: Any suggestions :)